src/test/java/com/epam/edumanagementsystem/EdumanagementsystemApplicationTests.java

```diff
@@ -6,8 +6,5 @@
 @SpringBootTest
 class EdumanagementsystemApplicationTests {
```

Comment:
Actually didn't get the meaning of this test class

Reply:
Neither did I